test/server.js

```diff
@@ -0,0 +1,12 @@
+import { main as startAppium } from 'appium';
+
+async function startServer (port, address, relaxedSecurityEnabled = false) {
+  // https://github.com/appium/appium/blob/2.0/packages/schema/lib/appium-config.schema.json
+  return await startAppium({
+    port,
+    address,
+    'relaxed-security': relaxedSecurityEnabled,
```

Comment:
kinda surprised this takes kebab instead of camel case version of the opts at this point.

Reply:
yeah, I'm not sure that's right.  I don't recall if it'll work offhand.  The parameter to `startAppium` (which is presumably the `main()` function) is an [`Args`](https://github.com/appium/appium/blob/2.0/packages/appium/types/cli.ts#L137) object, which is a union that includes `ServerArgs`--and all props in `ServerArgs` are camel-cased.

Reply:
Got it. Changed.